packages/vkui/src/components/ImageBase/ImageBase.tsx

```diff
@@ -111,6 +111,10 @@ export interface ImageBaseProps
    * Для корректной работы необходимо задать размеры хотя бы одной стороны картинки
    */
   keepAspectRatio?: boolean;
+  /**
+   * см. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/elementtiming
+   */
+  elementtiming?: string;
```

Comment:
Он должен называться `elementTiming`

```suggestion
  elementTiming?: string;
```

Reply:
<img width="320" alt="image" src="https://github.com/user-attachments/assets/53ae9952-4cb9-456f-b5a5-00accfc82765" />

так будет ругаться(

Reply:
Атрибут прокидывать надо `elementtiming`. Потом как c getFetchPriorityProp в зависимости от версии нужно будет прокидывать `elementtiming` или `elementTiming`, когда завезут поддержку в React

Reply:
аа пон, сейчас поменяю тип

Reply:
:zap: 583503e6f91002e7c79fb49534bfd705aa2621cf